tide/wall/resources/PowerTimer.qml

```diff
@@ -0,0 +1,49 @@
+import QtQuick 2.0
+import Tide 1.0
+
+import "qrc:/qml/core/style.js" as Style
+
+Item{
+    Rectangle {
+        id: countdown
+        anchors.fill: parent
+        color: Style.focusContextColor
+
+        states: [State {
```

Comment:
State { should be on next line. Use tools -> QML/JS -> Reformat File in QtCreator ;-)